__init__.py

```diff
@@ -1,3 +0,0 @@
-# This __init__ file is just here to support the
```

Comment:
I removed this file because I wasn't sure if this "installation" method will still work. Generally Python packages don't support use without installation.

Reply:
One usage of this library is as a git submodule, see for instance [archoffline](https://github.com/Torxed/archoffline) which has this as a submodule. Perhaps this is not needed in the long run now that it's a pypi project as well. I'll have to circle back, but I do like the ability to use it as a git submodule :)